<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RemoteDev::SftpSettingsWidget</class>
 <widget class="QWidget" name="RemoteDev::SftpSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="4" column="0">
      <widget class="QLabel" name="labelAuthType">
       <property name="text">
        <string>Authentication Type</string>
       </property>
       <property name="buddy">
        <cstring>comboAuthType</cstring>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QComboBox" name="comboAuthType">
       <property name="currentIndex">
        <number>1</number>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
       </property>
       <item>
        <property name="text">
         <string>Password</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Public Key</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Host-based</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Keyboard</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="labelHost">
       <property name="text">
        <string>Hostna&amp;me</string>
       </property>
       <property name="buddy">
        <cstring>editHost</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="editHost">
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelPort">
       <property name="text">
        <string>Port</string>
       </property>
       <property name="buddy">
        <cstring>spinPort</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="spinPort">
       <property name="maximum">
        <number>65535</number>
       </property>
       <property name="value">
        <number>22</number>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="labelUser">
       <property name="text">
        <string>Username</string>
       </property>
       <property name="buddy">
        <cstring>editUser</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="editUser">
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelTmout">
       <property name="text">
        <string>Timeout</string>
       </property>
       <property name="buddy">
        <cstring>spinTmout</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="spinTmout">
       <property name="suffix">
        <string> seconds</string>
       </property>
       <property name="minimum">
        <number>-1</number>
       </property>
       <property name="maximum">
        <number>360</number>
       </property>
       <property name="value">
        <number>30</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QStackedWidget" name="widgetAuthData">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="pagePasswd">
      <layout class="QFormLayout" name="formLayout_3">
       <item row="0" column="0">
        <widget class="QLabel" name="labelPasswd">
         <property name="text">
          <string>Password</string>
         </property>
         <property name="buddy">
          <cstring>editPasswd</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="editPasswd">
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
         <property name="clearButtonEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pagePubkey">
      <layout class="QFormLayout" name="formLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="labelKey">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>&amp;Key File</string>
         </property>
         <property name="buddy">
          <cstring>buttonFile</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="buttonFile">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="labelPhrase">
         <property name="text">
          <string>Passphrase</string>
         </property>
         <property name="buddy">
          <cstring>editPhrase</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="editPhrase">
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
         <property name="clearButtonEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>editHost</tabstop>
  <tabstop>spinPort</tabstop>
  <tabstop>spinTmout</tabstop>
  <tabstop>editUser</tabstop>
  <tabstop>comboAuthType</tabstop>
  <tabstop>buttonFile</tabstop>
  <tabstop>editPhrase</tabstop>
  <tabstop>editPasswd</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>comboAuthType</sender>
   <signal>activated(int)</signal>
   <receiver>widgetAuthData</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>193</x>
     <y>161</y>
    </hint>
    <hint type="destinationlabel">
     <x>219</x>
     <y>184</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
